webapp/security/views.py

```diff
@@ -47,7 +47,7 @@ def notice(notice_id):
             release_version = releases[codename]
             release_packages[release_version] = {}
             for package in pkgs:
-                if package.get("is_binary", False):
+                if not package.get("is_visible", True):
```

Comment:
Just to confirm that the intended change is if there is no `is_visible` we default to show the package?

Reply:
Yes. By default we show everything, and only if the `is_visible` this present and set to `false`. Then we hide it.